src/backend/src/routes/description-bullets.routes.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import { body } from 'express-validator';
+import { checkWorkPackageDescriptionBullet } from '../controllers/Description-Bullets.controllers';
+
+const descriptionBulletsRouter = express.Router();
+
+descriptionBulletsRouter.post(
+  '/check',
+  body('submitterId').isInt({ min: 0 }).not().isString(),
```

Comment:
you can use that int abstraction function you made in your first pr for these two lines

Reply:
Unfortunately that function hasn’t been implemented on the branch of develop that this branch is on, I think #107.